modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSMessageStoreImpl.java

```diff
@@ -1323,6 +1601,44 @@ public void deleteAllMessageMetadata(String storageQueueName) throws AndesExcept
             String task = RDBMSConstants.TASK_DELETING_METADATA_FROM_QUEUE + storageQueueName;
             close(preparedStatement, task);
             close(connection, task);
+            return deletedMessagecount;
+        }
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public int clearDlcQueue(String dlcQueueName) throws AndesException {
+
```

Comment:
Fixed in this class and some other identified classes by the PR https://github.com/wso2/andes/pull/403. Will progress onto the rest of the classes as well